docker/observation/.env.example

```
@@ -20,7 +20,7 @@ TEMPO_PORT=3200
 
 # Elasticsearch Configuration
 ELASTICSEARCH_PORT=9200
-ES_JAVA_OPTS="-Xms512m -Xmx512m"
+ES_JAVA_OPTS=-Xms512m -Xmx512m
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)